oak_runtime/src/node/http/mod.rs

```diff
@@ -166,6 +174,7 @@ impl HttpServerNode {
                 io::Error::new(io::ErrorKind::Other, format!("Incoming failed: {:?}", err))
             })
             .and_then(move |stream| {
```

Comment:
just checking: is it possible that it would be simpler to handle the issue at this level? have you already tried that?

Reply:
I did think of it but I cannot figure out how. 
It would be nice to call a `filter` method after `and_then` here. But there does not seem to be any such method. 
I also tried to implement something like https://github.com/hyperium/tonic/blob/04a8c0c82a4007f48c3bf3539a3f2312746fedd1/tonic/src/transport/server/incoming.rs#L18

but the types are too complicated, and there is no useful documentation. I would like to get something like `Result<TlsStream<TcpStream>, Error>` from `tokio_rustls::Accept`, and filter out the error cases, and only yield the ones that have a valid TlsStream. Let me know if you have any ideas for that conversion.